lib/sgtn-client/util/locale-util.rb

```diff
@@ -10,10 +15,29 @@ module SgtnClient
                   }
 
     class LocaleUtil
+
+
+        def self.get_best_locale(locale)
+            flocale = fallback(process_locale(locale))
+            flocale = SgtnClient::Config.configurations.default if flocale == get_source_locale()
```

Comment:
default maybe different from source_locale

Reply:
SgtnClient::Config.configurations.default is pointing to the local source.

Reply:
SgtnClient::Config.configurations.default doesn't mean the default locale here, it's just a var to point to the locale source.